registration/models.py

```diff
@@ -68,3 +68,21 @@ class Registration(models.Model):
 
     def __unicode__(self):
         return u"Registration for %s" % self.mom_msisdn
+
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .tasks import jembi_post_json
+
+
+@receiver(post_save, sender=Registration)
+def fire_jembi_post(sender, instance, created, **kwargs):
+    """ Send the registration info to Jembi.
+        For the clinic and chw registrations, fires an additional task that
+        uploads an XML document.
+    """
+    jembi_post_json.apply_async(kwargs={"registration_id": instance.id})
```

Comment:
missing `if created:` - currently it would send on edit too if someone opened it in Django admin or send a PATCH etc.